src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.cs

```diff
@@ -70,4 +70,17 @@ public sealed partial class RequiredAttributeAttribute : System.Attribute
         public RequiredAttributeAttribute(System.Type requiredContract) { }
         public System.Type RequiredContract { get { return default(System.Type); } }
     }
+    public static partial class IsPinned
+    {
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(8))]
```

Comment:
Do we commonly use the number values rather than the names?  e.g. I'd have expected to see:
```C#
[System.AttributeUsage(System.AttributeTargets.Struct)]
```